packages/ui-utils/src/mergeDeep.ts

```diff
@@ -66,25 +66,22 @@ function mergeSourceIntoTarget(target, source) {
         merged[key] = source[key]
       }
     })
-
     return merged
   } else {
     return { ...target }
   }
 }
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'item' implicitly has an 'any' type.
-function isObject(item) {
+function isObject(item: unknown) {
   return (
     item &&
     (typeof item === 'object' || typeof item === 'function') &&
     !Array.isArray(item)
   )
 }
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'item' implicitly has an 'any' type.
-function isArray(item) {
-  return item && Array.isArray(item)
```

Comment:
isn' the "item && " part needed? it is complaining about it not beeing a boolean, but maybe like this: `!!item && ...`?

Reply:
its not needed, thats why I deleted it :) Array.isArray(item) accepts `any`